### DISCUSSION TITLE
Making a keybinding to emulate `miw` (match inside word)?

### DISCUSSION THREAD

**Discussion by ajitid** [helix-editor/helix#9293] [Q&A] [answered]
I was thinking to somewhat emulate `*` of vim and I did find https://github.com/helix-editor/helix/discussions/5234#discussioncomment-5564487. However, rather than than doing `miw` it does does some navigation to select a word which doesn't always correctly select a word. I want to know if there's a proper way to do this, something along the lines of `:normal miw` or `["match_inner", ":keypress w"]`.

(Maybe I'm not searching properly, but I can't find a discussion/issue for it.)

**Comment by the-mikedavis** [comment, marked as answer]
That kind of binding would need macro keybinding (#1383). The comments you linked are workarounds since macro keybindings are not yet implemented

**Reply by ajitid** [reply]
Thank you, I'll subscribe to that issue
